roles/contiv_network/tasks/fwd_tasks.yml

```diff
@@ -0,0 +1,3 @@
+- name: set fwd mode
```

Comment:
I think we shall just have this task in `main.yml` than putting in a separate file. wdyt?